src/components/ArticleCard/index.tsx

```diff
@@ -0,0 +1,277 @@
+import * as React from 'react'
+import { css } from 'react-emotion'
+import Image, { ImageProps } from '../Image'
+import {
+  headlineFont,
+  storyListFont,
+  cardShadow,
+  regularFont,
+  bodyFont,
+} from '../../globals/mainsiteGlobalStyles'
+
+enum ContentType {
+  Tall = 'tall',
+  Long = 'long',
+  XL = 'longest',
+}
+
+interface ArticleCardProps {
+  type: string
+  title: string
+  content: string
+  imageurl: string
+  photographer: string
+  editor: string
+}
+
+export default class ArticleCard extends React.Component<ArticleCardProps> {
+  public static defaultProps = { imageurl: 'hi' }
+
+  public render() {
+    switch (this.props.type) {
```

Comment:
the jsx all inside the switch makes it a little hard to read, consider defining each type of card outside and referencing it once in the switch